ui/src/app/settings/containers/Settings/Settings.js

```diff
@@ -1,10 +1,26 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
 
+import actions from "app/settings/actions";
+import selectors from "app/settings/selectors";
 import Routes from "../../components/Routes";
 import Section from "app/base/components/Section";
 import SettingsNav from "../../components/Nav";
 
+const useFetchOnce = (fetchAction, loadedSelector) => {
```

Comment:
I've just copy-pasted this here, but will change to import from hooks.js once https://github.com/canonical-web-and-design/maas-ui/pull/61 has landed

Reply:
This should be imported from [`hooks.js`](https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/base/hooks.js) now.